stoplight-admin.gemspec

```diff
@@ -23,23 +23,23 @@ Gem::Specification.new do |gem|
     Dir.glob(File.join('lib', '**', '*.rb')) +
     Dir.glob(File.join('lib', '**', '*.haml'))
 
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 2.5.0'
 
   {
-    'haml' => '5.0.4',
-    'sinatra-contrib' => '2.2.3'
+    'haml' => '5.0',
```

Comment:
@gdeoliveira, the plan for this PR is to release a version with a bug fix and slightly bumped dependencies (`haml` to 5.x.x, `sinatra-contrib` to `3.0.x` and `stoplight` to `3.0.x`).
Would you happen to have any concerns about any particular dependency?

Reply:
We are currently in the process of upgrading Rails for a rather large application that is using `stoplight` v2. We are trying to be as conservative as possible in the amount of changes we introduce while doing so to minimize risk.

One of the dependencies we need to reach is (at least) `haml` v5.1, which this change addresses. At the same time we would like to avoid upgrading `stoplight` to v3 if we could help it.

What I was wondering is if besides releasing these bug fixes and dependency upgrades on version `0.4.0` you would consider releasing version `0.3.6` with this more flexible dependency on `haml` but without the `stoplight` version bump.

Reply:
Hey, @gdeoliveira, don't worry about stoplight 3.0. The only breaking change in this version is that. the minimally supported ruby version was changed from 2.5 to 2.6

Reply:
@gdeoliveira,
I agree with @bolshakov; there should not be any issues with stoplight 3.0.
Anyway, I updated dependencies for the intermediate `0.3.6` version.
Good luck with the Rails upgrade! I recommend checking those amazing articles by [Shopify](https://shopify.engineering/upgrading-shopify-to-rails-5-0), and [GitHub](https://github.blog/2018-09-28-upgrading-github-from-rails-3-2-to-5-2/) about the Rails upgrade. Hope they will help :)